framework/src/modules/http_api/index.js

```diff
@@ -24,7 +24,7 @@ class HttpAPIModule extends BaseModule {
 		return {
 			author: 'LiskHQ',
 			version: '0.1.0',
-			name: 'lisk-core-http-api',
+			name: 'lisk-framework-http-api',
```

Comment:
Please update this usage here as well, which was already wrong. 

https://github.com/LiskHQ/lisk-sdk/blob/3492bd749b40e3c2c5d615bd1e5e633d7da2b811/framework/src/controller/application.js#L338